config-model/src/main/java/com/yahoo/vespa/model/container/http/ssl/HostedSslConnectorFactory.java

```diff
@@ -42,7 +42,7 @@ private HostedSslConnectorFactory(Builder builder) {
         this.remotePortHeaders = List.copyOf(builder.remotePortHeaders);
         this.knownServerNames = Collections.unmodifiableSet(new TreeSet<>(builder.knownServerNames));
         builder.requestPrefixForLoggingContent.forEach(prefix -> {
-            var regex = "^.*:[0|1](\\.\\d+)?$";
+            var regex = "^.*:[0|1](\\.\\d+)?:\\d+[a-zA-Z]+$";
```

Comment:
```suggestion
            var regex = "^.*:[01](\\.\\d+)?:\\d+[a-zA-Z]+$";
```
This should be changed the other place too, from the previous PR!

Reply:
To ensure there are exactly _three_ parts, as expected in later code, perhaps also replace the leading `.*` with `[^:]*`.

Reply:
So ... this may fail during preparation, but is set in a feature flag? 
I think the flag should also enforce the pattern, then. Does it?